src/views/instances/details/deployment-summary.tsx

```diff
@@ -1,26 +1,47 @@
 import { FC } from 'react'
 import { Deployment } from '../../../types'
 import Moment from 'react-moment'
+import i18n from '@dhis2/d2-i18n'
 import { MomentExpiresFromNow } from '../../../components'
+import { IconClock16, IconUser16, IconUserGroup16, IconWarning16, IconClockHistory16, IconWorld16, IconInfo16 } from '@dhis2/ui'
 import styles from './deployment-summary.module.css'
 
-export const DeploymentSummary: FC<{ deployment: Deployment }> = ({ deployment }) => (
-    <dl className={styles.list}>
-        <dt>Name</dt>
-        <dd>{deployment.name || ' - '}</dd>
-        <dt>Description</dt>
-        <dd>{deployment.description || ' - '}</dd>
-        <dt>Group</dt>
-        <dd>{deployment.groupName || ' - '}</dd>
-        <dt>Created</dt>
-        <dd>{deployment.createdAt ? <Moment date={deployment.createdAt} fromNow /> : ' - '}</dd>
-        <dt>Updated</dt>
-        <dd>{deployment.updatedAt ? <Moment date={deployment.updatedAt} fromNow /> : ' - '}</dd>
-        <dt>Owner</dt>
-        <dd>{deployment.user?.email || ' - '}</dd>
-        <dt>public</dt>
-        <dd>{deployment.public === true ? 'Yes' : 'No'}</dd>
-        <dt>Expires</dt>
-        <dd>{deployment.ttl ? <MomentExpiresFromNow createdAt={deployment.createdAt} ttl={deployment.ttl} /> : ' - '}</dd>
-    </dl>
-)
+export const DeploymentSummary: FC<{ deployment: Deployment }> = ({ deployment }) => {
+
+    return (
+        <div className={styles.wrapper}>
+            <h2 className={styles.title}>{deployment.name || ' - '}</h2>
+            <hr className={styles.hr} />
```

Comment:
`hr`s should only be used when semantically correct (see [here](https://stackoverflow.com/a/9901454)).
In this case, it's there only for styling purposes, which can be done purely with CSS